package.json

```
@@ -11,6 +11,7 @@
         "deploy:docs": "gh-pages -d portal/public",
         "typecheck": "tsc --noEmit",
         "lint": "eslint '**/*.{js,jsx,ts,tsx}'",
+        "lint-fix": "eslint '**/*.{js,jsx,ts,tsx}' --fix",
```

```suggestion
        "lint:fix": "eslint '**/*.{js,jsx,ts,tsx}' --fix",
```

Følger bedre stilen i resten av fila. Men denne er ikke nødvendig, `eslint --fix` kjøres på alle filer i stage på commit.

Så hvordan du velger å formatere fila under utvikling er egentlig helt opp til deg. Anbefaller heller å sette opp en formatter/linter i IDEn din enn å kjøre denne, ganske trege kommandoen ( da denne parser alle filer i prosjektet). Siden den kjører på alle filer, så risikerer du endring i filer du ikke ønsker endring i.